test/paddedtests.jl

```diff
@@ -207,6 +207,7 @@ paddeddata(a::PaddedPadded) = a
             @test a .* v == v .* a
 
             @test n .+ n ≡ Vcat(2,Zeros{Int}(3))
+            @test n .+ v ≡ n .+ v
```

Comment:
I've added this one because currently this doesn't even hold on Julia 1.8-beta1. Testing this may save a lot of time in the future when trying to figure out why `n .+ v !== v .+ n`.